### DISCUSSION TITLE
Accessibility Features

### DISCUSSION THREAD

**Discussion by Potato314159** [SauceLLC/sauce4zwift#56] [Ideas]
Any thoughts on adding accessibility features for the blind? There was a feature request on the Zwift Riders Facebook page.

Just brainstorming but perhaps a tone that changes pitch as your 3s power drops above or below the current groups average power. Have that tone vary in frequency from once per second when you are off the back to a solid tone when you are at the front of the group.

Another beep could indicate an attack is happening.

There is some history of its use in other sports.

https://assistivetechnologyblog.com/2022/02/new-technology-action-audio-describes-sports-to-blind-audience.html

<img width="633" alt="PNG image" src="https://user-images.githubusercontent.com/2087131/203690296-70a35feb-2433-49b8-a209-df3e492932f7.png">

**Comment by mayfield** [comment]
Really interesting stuff.

What do you think about a voice to text system that would read status?   So
you could do some sonar type beeps and blips for immediate status to
indicate what's happening, say, in your particular group, but also have a
voice reading status every once in a while or when things are
starting/changing?

It might require a sighted person to set things up the first time
becaues I'm sure my ARIA is suboptimal right now, but then once it's
configured I would think it would be good to go for all future sessions.

I'm also wondering if there is a way for me to tap into an existing screen
reader technology.  I'll do a little research on what systems are currently
available and popular for this, because I might be able to just tap into an
already present voice to text system that the user is already using and
comfortable with.

Justin
